src/MovieDisplay/MovieDisplay.js

```diff
@@ -1,23 +1,59 @@
 import './MovieDisplay.css';
 import MovieDetail from '../MovieDetail/MovieDetail';
+import Header from '../Header/Header';
+import { fetchData } from '../util.js';
+import { Component } from 'react';
+import  ErrorDisplay from '../ErrorDisplay/ErrorDisplay';
+import { Link } from 'react-router-dom';
 
 
-const MovieDisplay = (props) => {
+class MovieDisplay extends Component {
```

Comment:
Converting this to a class component was the right move in our situation. When we learn Hooks, we should revisit this project and refactor to use those instead!